participants/yolo.md

```diff
@@ -0,0 +1,52 @@
+# <Project Name>
+
+## (0) Project Name and Team Slack Handles
+Our Project Name is Yolo.
+Our Team Slack Handles is UNTPFH4HW, and with Display name Max.
+
+## (1) Brief Project Description
+Yolo is a decentralized images dataset to develop ML and AI models based on Filecoin and IPFS. We are planning to storage [Open Images Dataset](https://storage.googleapis.com/openimages/web/index.html) on Filecoin and IPFS network by Powergate, and continue to enrich and improve it to be a decentralized dataset.
+
+## (2) Link to Project UI
+- [Yolo demo](https://yolo.coinsummer.io)
+- [Yolo front-end repository](https://github.com/CoinSummer/yolo)
+
+## (3) What does your application/UI do?
+Everyone can use Yolo dataset to train their own Machine-learning and AI models. Users who have images can also upload them to contribute the Yolo dataset together with us. There will be some token rewards (in the feature) for their contribution after these images been reviewed. The Yolo application works as following:
+![Yolo Network](https://raw.githubusercontent.com/CoinSummer/yolo/master/docs/yolo-network.png)
+
+## (4) If your project is using a curated dataset, which one are you using?
+We use [Open Images Dataset](https://storage.googleapis.com/openimages/web/index.html).
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+We are using a curated dataset.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We are planning to store at least 5TB data to the Filecoin network, maybe more than 10TB if everything goes well.
+
+## (7) How are you structuring the data?
+The images dataset are unstructured data with JPG or PNG format. However, we manage these dataset and our application data by database like MySQL.
+
+## (8) What pre-processing are you doing before ingesting the data?
+It is currently planned to package a series of images which have same type and category, and then store them to the Filecoin network by sending storage deals to miners. The size of storage deals will be 3GB to 5GB according to the network bandwidth when sending to miners.
```

Comment:
I believe there may currently be a technical limitation of 4GB for deals in Filecoin.  You're welcome to do larger deals subject to technical feasibility.

Reply:
Okey, we will try large size deals, get in touch.